lebedigital/raw_data_processing/metadata_extraction/mixture_metadata_extraction.py

```diff
@@ -0,0 +1,144 @@
+# Script for metadata-extraction: function takes in as argument a given path to
+# raw data and an output-path and returns a yaml-file with the metadata
+#
+# Workflow:
+# 1. Installing and importing the necessary libraries
+# 2. Defining a function to translate the labels from german to english
+# 3. Defining a function to extract metadata:
+# 3.a Look for sheets containing the word "Rezeptur" and load them
+# 3.b Reduce the dataframe to only necessary entries
+# 3.c Merge dataframe to a series
+# 3.d Export series as yaml-file
+#----------------------------------------------------------------------------------------------------------
+
+# In[1]: Setup
+
+from cmath import nan
+import pandas as pd
+import os
+import yaml
+
+def translate(
+        df, 
+        dic
+        ):
+
+    """
+        Translates german labels to english according to the translation.xls
+
+        Parameter
+        ---------
+        df
+            The dataframe with the german labels that should be translated.
+        dic
+            The excelfile that contains the translations, columns are "german" and "english"
+    """
+    
+    translation = pd.read_excel(dic)
```

Comment:
I think this is to simplify the reading in of the raw excel sheet (which cannot be changed) and to translate the file. However, I'm not sure why we need the translation of the orginal file. IMO, all the raw excel sheets are German, and I think it would be good to a have mapping such that when printing something, we could always change the printed output by a dictionary e.g. `translate[German,specimen]=Probekörper`, but internally I would only use a single (English) representation. That makes it much easier to work with that.

Reply:
No, this specifically is for the translation file: [translation.xlsx]( https://github.com/BAMresearch/LebeDigital/blob/unittest-mixture-metadata-extraction/lebedigital/raw_data_processing/metadata_extraction/translation.xlsx). If we want/need a translation file, I would prefer it to be not an exel file.

Reply:
Yes, you are right, that is correct, my mistake.